demos/chat-client/c++/assets.h

```diff
@@ -0,0 +1,3 @@
x61,0x72,0x65,0x61,0x27,0x3E,0x0A,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x3C,0x64,0x69,0x76,0x20,0x63,0x6C,0x61,0x73,0x73,0x3D,0x27,0x62,0x6F,0x74,0x2D,0x6D,0x73,0x67,0x27,0x3E,0x0A,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x3C,0x69,0x20,0x63,0x6C,0x61,0x73,0x73,0x3D,0x27,0x6D,0x61,0x74,0x65,0x72,0x69,0x61,0x6C,0x2D,0x69,0x63,0x6F,0x6E,0x73,0x20,0x62,0x6F,0x74,0x2D,0x69,0x6D,0x67,0x27,0x3E,0x66,0x61,0x63,0x65,0x3C,0x2F,0x69,0x3E,0x0A,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x3C,0x73,0x70,0x61,0x6E,0x20,0x63,0x6C,0x61,0x73,0x73,0x3D,0x27,0x6D,0x73,0x67,0x27,0x3E,0x45,0x6E,0x61,0x72,0x78,0x20,0x43,0x68,0x61,0x74,0x3C,0x2F,0x73,0x70,0x61,0x6E,0x3E,0x0A,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x3C,0x2F,0x64,0x69,0x76,0x3E,0x0A,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x3C,0x2F,0x64,0x69,0x76,0x3E,0x0A,0x0A,0x0A,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x3C,0x64,0x69,0x76,0x20,0x63,0x6C,0x61,0x73,0x73,0x3D,0x27,0x63,0x68,0x61,0x74,0x2D,0x69,0x6E,0x70,0x75,0x74,0x2D,0x61,0x72,0x65,0x61,0x27,0x3E,0x0A,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x3C,0x69,0x6E,0x70,0x75,0x74,0x20,0x74,0x79,0x70,0x65,0x3D,0x27,0x74,0x65,0x78,0x74,0x27,0x20,0x61,0x75,0x74,0x6F,0x66,0x6F,0x63,0x75,0x73,0x20,0x63,0x6C,0x61,0x73,0x73,0x3D,0x27,0x63,0x68,0x61,0x74,0x2D,0x69,0x6E,0x70,0x75,0x74,0x27,0x20,0x6F,0x6E,0x6B,0x65,0x79,0x70,0x72,0x65,0x73,0x73,0x3D,0x27,0x72,0x65,0x74,0x75,0x72,0x6E,0x20,0x67,0x69,0x76,0x65,0x6E,0x55,0x73,0x65,0x72,0x49,0x6E,0x70,0x75,0x74,0x28,0x65,0x76,0x65,0x6E,0x74,0x29,0x27,0x20,0x70,0x6C,0x61,0x63,0x65,0x68,0x6F,0x6C,0x64,0x65,0x72,0x3D,0x27,0x54,0x79,0x70,0x65,0x20,0x61,0x20,0x6D,0x65,0x73,0x73,0x61,0x67,0x65,0x20,0x2E,0x2E,0x2E,0x27,0x20,0x61,0x75,0x74,0x6F,0x63,0x6F,0x6D,0x70,0x6C,0x65,0x74,0x65,0x3D,0x27,0x6F,0x66,0x66,0x27,0x3E,0x0A,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x3C,0x62,0x75,0x74,0x74,0x6F,0x6E,0x20,0x63,0x6C,0x61,0x73,0x73,0x3D,0x27,0x63,0x68,0x61,0x74,0x2D,0x73,0x75,0x62,0x6D,0x69,0x74,0x27,0x3E,0x3C,0x69,0x20,0x63,0x6C,0x61,0x73,0x73,0x3D,0x27,0x6D,0x61,0x74,0x65,0x72,0x69,0x61,0x6C,0x2D,0x69,0x63,0x6F,0x6E,0x73,0x27,0x3E,0x73,0x65,0x6E,0x64,0x3C,0x2F,0x69,0x3E,0x3C,0x2F,0x62,0x75,0x74,0x74,0x6F,0x6E,0x3E,0x0A,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x3C,0x2F,0x64,0x69,0x76,0x3E,0x0A,0x0A,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x20,0x3C,0x2F,0x64,0x69,0x76,0x3E,0x0A,0x20,0x20,0x20,0x20,0x3C,0x2F,0x64,0x69,0x76,0x3E,0x0A,0x20,0x20,0x20,0x20,0x3C,0x73,0x63,0x72,0x69,0x70,0x74,0x20,0x73,0x72,0x63,0x3D,0x22,0x2E,0x2F,0x73,0x63,0x72,0x69,0x70,0x74,0x2E,0x6A,0x73,0x22,0x3E,0x3C,0x2F,0x73,0x63,0x72,0x69,0x70,0x74,0x3E,0x0A,0x0A,0x3C,0x2F,0x62,0x6F,0x64,0x79,0x3E,0x0A,0x0A,0x3C,0x2F,0x68,0x74,0x6D,0x6C,0x3E};
```

Comment:
There should be a comment at the top of this file saying that this is auto-generated and pointing to the source.
Could you add the script to the repo as well? We don't have to make it a part of the build pipeline, but there should be a way to reproduce this